### ISSUE TITLE
Topic: deprecate offline parameter validation and "combinations.json"

### ISSUE THREAD

**Issue by pzhlkj6612** [miurahr/aqtinstall#748]
**Is your feature request related to a problem? Please describe.**

Let's say goodbye to our old friend "Specified Qt version is unknown" and their partners. They always confuse our users.

```console
$ aqt install-qt linux desktop 9.9.9
INFO    : aqtinstall(aqt) v3.1.11 on Python 3.10.12 [CPython GCC 11.4.0]
WARNING : Specified Qt version "9.9.9" did not exist when this version of aqtinstall was released. This may not install properly, but we will try our best.
WARNING : Failed to download checksum for the file 'Updates.xml'. This may happen on unofficial mirrors.
ERROR   : Failed to locate XML data for Qt version '9.9.9'.
==============================Suggested follow-up:==============================
* Please use 'aqt list-qt linux desktop' to show versions available.

```

```console
$ aqt install-qt linux desktop 9.9.9
INFO    : aqtinstall(aqt) v2.2.4 on Python 3.10.12 [CPython GCC 11.4.0]
WARNING : Specified Qt version is unknown: 9.9.9.
ERROR   : Failed to locate XML data for Qt version '9.9.9'.
==============================Suggested follow-up:==============================
* Please use 'aqt list-qt linux desktop' to show versions available.

```

```console
$ aqt install 9.9.9 linux desktop
aqtinstall(aqt) v1.2.5 on Python 3.10.12 [CPython GCC 11.4.0]
Specified Qt version is unknown: 9.9.9.
Download error when access to https://download.qt.io/online/qtsdkrepository/linux_x64/desktop/qt9_999/Updates.xml
Server response code: 404, reason: Not Found

```

**Describe the solution you'd like**

```console
$ aqt install 9.9.9 linux desktop
aqtinstall(aqt) v...
WARNING : Failed to download checksum for the file 'Updates.xml'. This may happen on unofficial mirrors.
ERROR   : Failed to locate XML data for Qt version '9.9.9'.
============================================================
Available Qt versions: 5.15.2, 6.0.4...

```

**Describe alternatives you've considered**

None.

**Additional context**

From [What's the purpose of making the "combinations.json" file? · miurahr/aqtinstall · Discussion #747](https://github.com/miurahr/aqtinstall/discussions/747).

**Comment by pzhlkj6612** [comment]
Solved by #812.

**Comment by pzhlkj6612** [comment]
Another example: https://github.com/jurplel/install-qt-action/pull/251#issuecomment-2402928193 :

> WARNING : Specified Qt version "6.8.0" did not exist when this version of aqtinstall was released. This may not install properly, but we will try our best.
> 
> Qt 6.8.0 seems to be not detected.
> ... link ...
> 
> Qt 6.8.0 was released

And please don't make me wrong, I'm not going to blame anyone. This is an example to show how those warnings confused users.
